test.js

```diff
@@ -5,6 +5,8 @@ const proxyquire = proxyquireBase.noPreserveCache().noCallThru();
 
 const expectedFallback = 'en-US';
 
+const execaImport = './execa';
```

Comment:
Couldn't you use `execa` from devDependencies just for testing?

Reply:
This file isn't actually importing any modules. `proxyquire` mocks the import that has this path in `index.js`